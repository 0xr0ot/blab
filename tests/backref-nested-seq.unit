"0" \("1" \("2" \("3"\) \("4"\)\)\) \("5"\) 
#    |     |     |    |  |    | | |  '----|
#    |     |     |    |  |    | | |       '-> 5 = 5
#    |     |     |    |  '----| | |
#    |     |     |    |       '-------------> 4 = 4
#    |     |     '----|         | |
#    |     |          '---------------------> 3 = 3
#    |     '--------------------| |
#    |                          '-----------> 2 = 23
#    '----------------------------|
#                                 '---------> 1 = 1234
   "\n" \1
   "\n" \2 
   "\n" \3
   "\n" \4
   "\n" \5

