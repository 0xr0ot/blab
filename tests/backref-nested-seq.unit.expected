012345
1234
234
3
4
5